packages/account-v2/package.json

```diff
@@ -27,8 +27,10 @@
         "react-router-dom": "^5.2.0",
         "react-transition-group": "4.4.2",
         "tailwind-merge": "^1.14.0",
+        "ua-parser-js": "^1.0.37",
         "usehooks-ts": "^2.7.0",
-        "yup": "^0.32.11"
+        "yup": "^0.32.11",
+        "dayjs": "^1.8.15"
```

Comment:
Can you if this is the latest package? I can see this `1.11.10`  as the latest version [here](https://www.npmjs.com/package/dayjs)